api/docs/docs/overview.md

```diff
@@ -24,17 +24,36 @@ make sure to keep you informed on forthcoming changes.
+
+- A comprehensive compilation of the latest U.S. state and county COVID data, including cases, hospitalizations, hospital capacity, hospital utilization (by both COVID and non-COVID patients), deaths, testing, and tracer numbers.
+
+  This data is aggregated from the best-available official, public and private sources; it is secured and vetted by our dedicated team and advisory board of public health, epidemiology, public policy experts. When necessarily, the data is scraped directly from official state and county COVID dashboards, and verified against other sources for veracity.
+
+- COVID data that is regularly verified, sanitized, and updated, generally on a daily-basis, by our team of engineers, modelers, and analysts.
+
+- An R<sub>t</sub> (infection growth rate) that is optimized for capturing COVID dynamics at the county-level. <!-- Read more about how it’s calculated here. -->
+
+- Our simple yet robust green-yellow-orange-red [COVID risk-level framework](https://globalepidemics.org/wp-content/uploads/2020/09/key_metrics_and_indicators_v5-1.pdf), researched and developed in conjunction with the Harvard Global Health Institute (HGHI) and Harvard Edmund J. Safra Center for Ethics.
 
 ### License
 
 Data is licensed under [Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International](https://creativecommons.org/licenses/by-nc-nd/4.0/).
 
-Under this new licensing arrangement, all commercial entities wishing to access and download Covid Act Now's
-data must contact <info@covidactnow.org> to acquire a license. We define commercial users as any individual
-or entity that sells goods or services for the purposes of making a profit. 
+We are a volunteer-driven 501(c)3 non-profit whose mission is to provide reliable and trustworthy COVID data,
+with a focus on U.S. counties. Since we started on March 20th, we’ve made our model and data open-source
+and freely available to the public.
+
+This data requires a non-trivial amount of work to organize, sanitize, maintain, and develop,
+especially given the fast-changing nature of COVID and health data in the U.S. In order to
+ensure that we can continue to deliver best-in-class open-source COVID data, our tiny,
+volunteer-driven team appreciates donations here.
+
+We also ask that, as legally required by our license, all commercial entities wishing to
+access our API contact <info@covidactnow.org> to acquire a commercial license. We define
+commercial users as any individual or entity engaged in commercial activities, such as selling goods or services.
 
-Under this license, our non-commercial users can still freely download, use, share, modify, or build upon our source code.
+Our non-commercial users can freely download, use, share, modify, or build upon our source code.
```

Comment:
I'm not a lawyer, but surely we want non-commercial users to only distribute data with the same terms (free for non-commercial users, commercial entities need to acquire a licence). IE https://creativecommons.org/licenses/by-nc-sa/4.0/?ref=chooser-v1